raiden-dapp/src/router/guards/connection.ts

```diff
@@ -1,26 +1,23 @@
 import { RouteNames } from '../route-names';
 import type { NavigationGuardChild } from './types';
 
-const accountRoutes = [
+const freeAccountRoutes = [
```

Comment:
`unprotectedAccountRoutes`?

Reply:
I was so unhappy with the naming. Thank you very much for the suggestion. 🙏🏾

Reply:
✔️